scrunch/datasets.py

```diff
@@ -105,11 +105,10 @@ def _get_connection(file_path='crunch.ini'):
             "Unable to find crunch session, crunch.ini file or environment variables.")
 
 
-def get_dataset(dataset, connection=None, editor=False, project=None):
```

Comment:
It wasnt' the intention of the ticket to remove the project parameter. We should keep it for the case when we pass a dataset by name and need a project to clarify which we are searching for (dataset names are not globally unique).